build/widgets/table.js

```diff
@@ -52,25 +50,25 @@ parseOrdering = function(ordering, data) {
 };
 
 getAlias = function(ordering, column) {
-  return _.result(_.findWhere(ordering, {
+  return _.result(_.find(ordering, {
     name: column
   }), 'alias');
 };
 
 normalizeTitle = function(title) {
-  return _.str.underscored(title).toUpperCase().replace(/_/g, ' ');
+  return _.trim(title).replace(/([a-z\d])([A-Z]+)/g, '$1 $2').replace(/[_-\s]+/g, ' ').toUpperCase();
```

Comment:
`_.str.underscored` was not quite the same as `_.snakeCase`, and since this function only used that as a proxy for normalization I instead merged it together and skipped a bunch of wasted steps